### DISCUSSION TITLE
域名frp-any.top如何申请https证书？

### DISCUSSION THREAD

**Discussion by chouqiao** [natfrp/wiki#387] [使用问题]
没有域名所有权，无法验证cname，我应该如何申请frp-any.top的https证书？我用于群晖docke容器的外网访问，请指导，谢谢

**Comment by Mr-XiaoYi** [comment]
它的域名只是让你提交重定向解析，应该并不让你使用，就想别人的cdn节点一样，cdn节点禁止访问

**Reply by Mr-XiaoYi** [reply]
你可以尝试购买域名，打折的很多，或者找一个二级域名分发。

**Reply by chouqiao** [reply]
那我应该怎么用我的域名指向这个sakura提供的隧道

**Reply by LTHPKBTE** [reply]
cname

**Reply by QLozin** [reply]
你只需要用你的域名通过CNAME指向frp的域名，同时给frp配置你的域名证书就好了（不过需要带端口访问，目前还没有找到解决方案）
